oat-sa/extension-tao-deliver-connect
# UNO-229: Publish aliased metadata

Requires [`tao`](https://github.com/oat-sa/tao-core/pull/2503).

- [UNO-229](https://oat-sa.atlassian.net/browse/UNO-229) feature: publish aliased test meta properties to a remote delivery automatically
- [UNO-229](https://oat-sa.atlassian.net/browse/UNO-229) chore(dependency): depend on `generis` `12.21.0`, which introduced `GenerisRdf::PROPERTY_ALIAS`

In order to test:

1. Create a new property on a class level
2. Assign that property some `Alias` value via `Manage Schema`
3. Create a new test under that class
4. Set some value to the test property
5. Publish that test to a remote deliver service
6. Query the deliver service to check if the published delivery has the metadata set (`/api/v1/deliveries/<remote_delivery_id>`)
